### DISCUSSION TITLE
Thank you for your work on new documentation on DDD best practices

### DISCUSSION THREAD

**Discussion by Stirda** [abpframework/abp#6395] [General]
I see a lot of work on documentation in recent commits (again since two days). I appreciate especially work on "domain vs application" responsability documentation by @hikalkan . In my company (and our clients) I give hints on how to best follow DDD/ABP principles. Read your recent work is as cool as I would talking to this with you and receive your expert view of these concepts. Really appreciate 🙏

**Comment by vnetonline** [comment]
Yes a very good read
